### ISSUE TITLE
snap version 1.2.9 broken

### ISSUE THREAD

**Issue by mike-geiger** [dceejay/nodered.snap#20]
When upgraded to 1.2.9 Node Red is giving Connection refused in the browser. As a work around I have reverted to the previous version "snap revert node-red" which restored functionality

**Comment by mike-geiger** [comment]
i see the following in the logs.
```
2021-02-10T16:00:16Z systemd[1]: Started Service for snap application node-red.node-red.
2021-02-10T16:00:17Z node-red.node-red[17781]: NOW STARTING NODE-RED : /snap/node-red/671/bin/node-red -u /root/snap/node-red/671 on port 1880
2021-02-10T16:00:17Z node-red.node-red[17781]: 10 Feb 08:00:17 - [error] Failed to start server:
2021-02-10T16:00:17Z node-red.node-red[17781]: 10 Feb 08:00:17 - [error] Error: EPERM: operation not permitted, copyfile '/root/snap/node-red/671/.config.users.json' -> '/root/snap/node-red/671/.config.users.json.backup'
```

**Comment by dceejay** [comment]
the stable channel should be OK again now (reverted to node10) - issue is with underlying snapd - fix will be in their 2.49.1
